packages/cli/test/integration/credentials.controller.test.ts

```diff
@@ -22,6 +22,201 @@ beforeEach(async () => {
 
 type GetAllResponse = { body: { data: ListQuery.Credentials.WithOwnedByAndSharedWith[] } };
 
+describe('POST /credentials', () => {
+	const testPayload = {
+		name: 'test',
+		type: 'test',
+		data: { super: 'secret' },
+		nodesAccess: [],
+	};
+
+	describe('as unauthorized user', () => {
+		it('returns 401', async () => {
+			await testServer.authlessAgent.post('/credentials').send(testPayload).expect(401);
+		});
+	});
+
+	describe('as a member', () => {
```

Comment:
The `as a member` and `as owner` tests look identical to me. This would be a good place to use `describe.each` instead of duplicating the test code.

Reply:
Sure, the only difference is the user making the requests. I can use `each` for that. 

Over time I've grown to like dead simple tests. Adding abstractions is a slippery slope.

But I think a `.each` is innocuous enough.